pkgs/by-name/ke/keydb/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libuuid,
+  curl,
+  pkg-config,
+  withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd,
+  systemd,
+  tlsSupport ? !stdenv.hostPlatform.isStatic,
+  openssl,
+  jemalloc,
+  which,
+  tcl,
+  tcltls,
+  ps,
+  getconf,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keydb";
+  version = "6.3.4";
+
+  src = fetchFromGitHub {
+    owner = "snapchat";
+    repo = pname;
```

Comment:
```suggestion
    repo = "keydb";
```

Reply:
is this now the preferred way of doing things?

Reply:
Mostly yes, see https://github.com/NixOS/nixpkgs/issues/277994. My understanding is that despite using `rec` or not, `repo = pname` doesn't carry any true associativity (i.e. when one changes, the referenced value should also change). And it's even less intuitive to view.

Reply:
ah i see, thanks for the input :)

Reply:
Why is this resolved?